ui/src/pages/Dashboard/dashboardStyle.css

```diff
@@ -328,3 +328,95 @@
     grid-row: 3 / 6;
     margin-top: 16px;
 }
+
+/* resource tab */
+
+#resource_tab_label_viz {
+    min-width: 450px;
+}
+
+[data-input-id='resource_tab_input'] {
+    grid-column: 6;
+    align-content: flex-end;
+    grid-row: 1;
+    place-self: end;
+    margin-top: 32px;
+
+    [data-test='input-container'] {
+        display: inline-flex;
+        padding: 0;
+    }
+    [data-test='input-title-remove-container'] {
+        margin: 5px;
+    }
+    [data-test='input-title'] {
+        font-size: 14px;
+    }
+}
+
+#resource_tab_description_viz {
+    grid-column: 1 / span 4;
+    grid-row: 1;
+    height: fit-content;
+    margin-top: 32px;
+}
+
+#resource_tab_timerange_label_start_viz {
+    grid-column: 1;
+    grid-row: 1;
+    inline-size: fit-content;
+    width: 145px;
+    margin-top: 35px;
+
+    [data-test='majorValue'] {
+        font-size: 14px;
+        line-height: 14px;
+        font-family: 'Splunk Platform Sans', 'Splunk Platform Sans', 'Proxima Nova', Roboto, Droid,
+            'Helvetica Neue', Helvetica, Arial, sans-serif;
+        font-weight: 400;
+        color: rgb(108, 118, 133);
+    }
+    [data-test='action-menu-wrapper'] {
+        display: none;
+    }
+}
+
+#resource_tab_timerange_label_end_viz {
+    grid-column: 1;
+    grid-row: 1;
+    inline-size: fit-content;
+    margin-left: 145px;
+    width: 145px;
+    display: flex;
+    margin-top: 35px;
+
+    [data-test='majorValue'] {
+        font-size: 14px;
+        line-height: 14px;
+        font-family: 'Splunk Platform Sans', 'Splunk Platform Sans', 'Proxima Nova', Roboto, Droid,
+            'Helvetica Neue', Helvetica, Arial, sans-serif;
+        font-weight: 400;
+        color: rgb(108, 118, 133);
+    }
+    [data-test='action-menu-wrapper'] {
+        display: none;
+    }
+}
+
+#resource_tab_timerange_label_end_viz::before {
+    content: '- ';
+    margin: auto;
+    color: rgb(108, 118, 133);
```

Comment:
please extract to the variable

Reply:
12 occurrences
![image](https://github.com/splunk/addonfactory-ucc-generator/assets/142901247/a2c21f69-e3a7-4b29-90eb-05b2aaa94d4f)

Reply:
moved to variable, also extracted some more shared styles to single reference.